src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -39,18 +39,7 @@ private string GetDebuggerDisplay()
         private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
-        protected NodeFlags flags
-        {
-            get => _nodeFlagsAndSlotCount.NodeFlags;
-            set => _nodeFlagsAndSlotCount.NodeFlags = value;
-        }
-
-        /// <inheritdoc cref="NodeFlagsAndSlotCount.SmallSlotCount"/>>
-        private byte _slotCount
-        {
-            get => _nodeFlagsAndSlotCount.SmallSlotCount;
-            set => _nodeFlagsAndSlotCount.SmallSlotCount = value;
-        }
+        private NodeFlags Flags => _nodeFlagsAndSlotCount.NodeFlags;
```

Comment:
this is now private.  No need for anything outside of this type to access this value.  All functionality is already exposed through clearer properties, or helper methods.